src/primitives/transaction.h

```diff
@@ -1162,10 +1067,15 @@ struct CMutableTransaction
             if (isOverwinterV3 || isSaplingV4 || isFuture) {
                 READWRITE(nExpiryHeight);
             }
-            if (isSaplingV4 || isFuture) {
```

Comment:
I don't recall why `isFuture` was part of this case. Not related to this PR, but it's odd and we should revisit and figure out why.

Reply:
Probably the "future" format was added when v4 was the latest format, and then never updated to v5.